### PR TITLE
Fix Cypress Tests

### LINKED ISSUE #532: Cypress Tests Failure
Cypress Tests are failing in the github actions and It seems to be blocking most of the PRs right now in repository.

There has been some initial discussion at https://github.com/publiclab/leaflet-environmental-layers/pull/529

Specifically Purple Air Marker Layer and Owm Layer tests are failing. 
image

Maybe related to #360 ?

Noting https://github.com/publiclab/leaflet-environmental-layers/pull/529#issuecomment-870934595, the tests initially passed when the actions were setup

### PR DESCRIPTION
**Pull request by daemon1024**
Fixes #532

### CODE REVIEW

**Comment by daemon1024** [comment]
image
 Fixed one of the tests 🥳

**Comment by daemon1024** [comment]
PurpleAirMarker Tests Fixed 🥳 🥳 

Giving some context on what I did,

image

the popup content seemed to be checking for label value which is very specific, instead I checked if it has the data provider url at the end since that will constant in every purple air marker popup :)

**Comment by daemon1024** [comment]
Everything passes. pheww. 😸

File: cypress/integration/owm.spec.js [hunk 1]
```diff
@@ -4,17 +4,22 @@ describe('OpenWeatherMap layer', function() {
     cy.wait(300)
     cy.window().then((win) => {
       cy.get('.leaflet-tile-pane').children().should('have.length', 1)
-      cy.get('#map-city label').click({ force: true })
+      cy.get('#map-Clouds label').click({ force: true })
```
Comment (jywarren) [inline cypress/integration/owm.spec.js:7]: Nice - will this be more stable? Like, there are clouds everywhere even though there aren't cities everywhere?

File: cypress/integration/owm.spec.js [hunk 2]
```diff
@@ -4,17 +4,22 @@ describe('OpenWeatherMap layer', function() {
...
       cy.get('.leaflet-tile-pane').children().should('have.length', 2)
-      cy.get('.img')
+      cy.get('img')
```
Comment (jywarren) [inline cypress/integration/owm.spec.js:9]: ah, here, it's just going to find ANY image. Let's make it a bit more specific. What is something the clouds layer is DEFINITELY going to display and does it have any characteristic CSS? Does it appear in a named layer?

File: cypress/integration/purpleAirMarker.spec.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ describe('Purple layer - markers', function() {
     const spy = cy.spy(window.top.aut.PurpleAirMarkerLayer, 'requestData')
     cy.get('[title="Show minimal markers"]').click().then(() => {
       expect(spy).to.be.called
-      cy.get('.leaflet-overlay-pane svg g').children().should('have.length', 2)
+      cy.get('.leaflet-overlay-pane svg g').children().should('have.length.gt', 0)
```
Comment (jywarren) [inline cypress/integration/purpleAirMarker.spec.js:35]: Great!

File: cypress/integration/purpleAirMarker.spec.js [hunk 4]
```diff
@@ -43,7 +43,7 @@ describe('Purple layer - markers', function() {
   it('shows popup', function() {
     cy.get('.leaflet-overlay-pane svg g').children().last().click({ force: true })
     cy.get('.leaflet-popup-pane').children().should('have.length', 1)
-    cy.get('.leaflet-popup-content').should('contain', 'MandMnorth40')
+    cy.get('.leaflet-popup-content').should('contain', 'www.purpleair.com')
```
Comment (jywarren) [inline cypress/integration/purpleAirMarker.spec.js:46]: Excellent.

**Review by jywarren — CHANGES_REQUESTED** [review]
I think just one change on the "img" selector! Great work!

File: cypress/integration/owm.spec.js (hunk 1 above)
Comment (daemon1024) [reply cypress/integration/owm.spec.js:7]: Yeah clouds,temp and pressure seemed to be more general to me among other maps. I just went with clouds.

File: cypress/integration/owm.spec.js (hunk 2 above)
Comment (daemon1024) [reply cypress/integration/owm.spec.js:9]: The tiles seem to contain these images

``

So check if the image src contains the term clouds?

**Comment by jywarren** [comment]
Or openweathermap.org/map/clouds !

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Excellent!!! OK, merging!

**Pull request merged**